subscribie/blueprints/admin/__init__.py

```diff
@@ -681,14 +681,26 @@ def formatted_status(stripe_external_id):
 @login_required
 def subscribers():
     page = request.args.get("page", 1, type=int)
+    action = request.args.get("action")
 
-    people = (
-        database.session.query(Person)
-        .order_by(desc(Person.created_at))
-        .paginate(page=page, per_page=5)
-    )
+    show_active = action == "show_active"
```

Comment:
The people query is always ran, sometimes twice if `show_active` is set.

Only one query should run (if/else show active)

(It's querying the database twice unnecessarily)

Reply:
Line 691, 694

Reply:
@schumskie I've resolved the above and pushed to this branch :+1:  https://github.com/Subscribie/subscribie/pull/424/commits/a0d36d5c669555dc37e84ac0f2d13bdee3e2bbee